raster/src/test/scala/geotrellis/raster/reproject/ReprojectSpec.scala

```diff
@@ -283,5 +283,13 @@ class ReprojectSpec extends FunSpec
 
       windowedReproject.rasterExtent should be (regularReproject.rasterExtent)
     }
+
+    it ("should reproject cea projection into WebMercator correctly") {
+      val geoTiff = SinglebandGeoTiff(geoTiffPath("reproject/cea.tif"))
+      val raster = geoTiff.raster.reproject(geoTiff.crs, WebMercator)
+
+      geoTiff.crs.toProj4String should be ("+proj=cea +lat_ts=33.75 +lon_0=-117.333333333333 +x_0=0.0 +y_0=0.0 +datum=NAD27 +units=m ")
+      raster.extent should be (geoTiff.extent.reproject(geoTiff.crs, WebMercator))
```

Comment:
This test would fail, is it expected? @echeipesh @lossyrob 

```
- should reproject cea projection into WebMercator correctly *** FAILED ***
[info]   Extent(-1.308996986113024E7, 5120515.980949544, -1.3059127661588239E7, 5176329.847734357) was not equal to Extent(-1.3089964868731003E7, 5120540.114447579, -1.3059110371614724E7, 5176329.847734357) (ReprojectSpec.scala:294)
```